### PR TITLE
fix subSahara QA

### PR DESCRIPTION
**Pull request by divinecharlotte**
fixed the given QA reviews on this section

### CODE REVIEW

File: src/sections/subSahara.html [hunk 1]
```diff
@@ -8,52 +8,54 @@
...
-<p class="font-railwayBold text-xs text-white">sehr gutes <br> Investitionsklima</p>
-    </div>
-    <div class="flex gap-10">
-        <img src="../../images/tick.svg" width="90"  alt="people" >
-<p class="font-railwayBold text-xs text-white">ausgezeichnete Infrastruktur <br> und hoher Digitalisierungsgrad</p>
-    </div>
-</footer>
-</section>
+        
+    <footer class="flex lg:flex-row flex-col  justify-between items-center lg:pr-48 h-80 p-2 xl:bg-no-repeat lg:pl-36 " style="background-image: url(../../images/bg_hero-1.webp)">
```
Comment (ishimwe07) [inline src/sections/subSahara.html]: below the footer there is no background color of blue, it's just oblique line below the linear gradient.


image

**Review by ishimwe07 — CHANGES_REQUESTED** [review]
On my larger screen you can add the background colors to the whole page.


image

**Review by NelHim — APPROVED** [review]
1. Try to center only the content, not the background on very large screens
2. Make the bottom oblique

**Review by NelHim — CHANGES_REQUESTED** [review]
1. Try to center only the content, not the background on very large screens
2. make sure your top and bottom are oblique

**Review by NoellaPromise — APPROVED** [review]
Good Job!

**Review by NelHim — APPROVED** [review]

**Comment by NelHim** [comment]
Good Job!

**Review by ishimwe07 — APPROVED** [review]
Looks Good now.

**Pull request merged**